pygmt/figure.py

```diff
@@ -167,8 +174,40 @@ def psconvert(self, **kwargs):
             using the input names as base, which are appended with an
             appropriate extension. Use this option to provide a different name,
             but without extension. Extension is still determined automatically.
-        icc_gray : bool
-            Enforce gray-shades by using ICC profiles.
+        resize : str
+            [**+m**\ *margins*][**+s**\ [**m**]\ *width*\
+            [/\ *height*]][**+S**\ *scale*] ].
+            Adjust the BoundingBox and HiResBoundingBox by scaling and/or
+            adding margins. Append **+m** to specify extra margins to extend
+            the bounding box. Give either one (uniform), two (x and y) or four
+            (individual sides) margins; append unit [Default is set by
+            :term:`PROJ_LENGTH_UNIT`]. Append **+s**\ *width* to resize the
+            output image to exactly *width* units. The default unit is set
+            by :term:`PROJ_LENGTH_UNIT` but you can append a new unit and/or
+            impose different width and height (**Note**: This may change the
+            image aspect ratio). What happens here is that Ghostscript will do
+            the re-interpolation work and the final image will retain the DPI
+            resolution set by ``dpi``.  Append **+sm** to set a maximum size
+            and the new *width* is only imposed if the original figure width
+            exceeds it. Append /\ *height* to also impose a maximum height in
+            addition to the width. Alternatively, append **+S**\ *scale* to
+            scale the image by a constant factor.
+        bb_style : str
+            Set optional BoundingBox fill color, fading, or draw the outline
+            of the BoundingBox. Append **+f**\ *fade* to fade the entire plot
+            towards black (100%) [no fading, 0]. Append **+g** \*paint* to
+            paint the BoundingBox behind the illustration and append **+p**\
+            [*pen*] to draw the BoundingBox outline (append a pen or accept
+            the default pen of 0.25p,black). Note: If both **+g** and **+f**
+            are used then we use paint as the fade color instead of black.
+            Append **+i** to enforce gray-shades by using ICC profiles.
+            Ghostscript versions >= 9.00 change gray-shades by using ICC
+            profiles. Ghostscript 9.05 and above provide the
+            '-dUseFastColor=true' option to prevent that and that is what
+            **psconvert** does by default, unless modifier **+i** is set.
+            Note that for Ghostscript >= 9.00 and < 9.05 the gray-shade
+            shifting is applied to all but PDF format. We have no solution to
+            offer other than suggesting you upgrade Ghostscript.
```

Comment:
I think it's OK to remove this long description about old ghostscript versions, because these versions are [~10 years old](https://www.ghostscript.com/doc/9.55.0/History9.htm), and I believe very few people will use them.  
```suggestion
```